Resources/Locale/en-US/ui/options.ftl

```diff
@@ -79,7 +79,7 @@ ui-options-function-open-character-menu = Open character menu
 ui-options-function-open-context-menu = Open context menu
 ui-options-function-open-crafting-menu = Open crafting menu
 ui-options-function-open-inventory-menu = Open inventory
-ui-options-function-open-tutorial = Open tutorial
+ui-options-function-open-info = Open server info
```

Comment:
did you change the corresponding Loc.GetString?

Reply:
Yes